### PR TITLE
Remove subscriptions

### PR DESCRIPTION
**Pull request by nilscox**

### CODE REVIEW

File: apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts [hunk 1]
```diff
@@ -1,125 +1,93 @@
...
-    const comment = await this.commentRepository.getComment(commentId);
-    assert(comment);
+  async handle({ entityId: eventId, commentId, authorId }: EventCommentCreated): Promise<void> {
+    const event = defined(
+      await this.database.db.query.events.findFirst({
+        where: eq(schema.events.id, eventId),
+        with: {
+          organizer: true,
+          comments: true,
+          participants: true,
```
Comment (Bopzor) [inline apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts:33]: **suggestion**

do not include participant with negative answer

File: apps/server/src/events/event.e2e.spec.ts [hunk 2]
```diff
@@ -63,10 +63,10 @@ describe('[E2E] Request', () => {
 
     expect(await test.fetch(`/events/${eventId}`, { token })).toHaveProperty('body.body', '<p>body</p>');
 
-    expect(await test.application.getMemberNotifications({ memberId: test.member.id })).toHaveProperty(
-      'notifications.0',
-      expect.objectContaining<Partial<shared.Notification>>({ title: 'Nouvel événement', content: 'title' }),
-    );
+    // expect(await test.application.getMemberNotifications({ memberId: test.member.id })).toHaveProperty(
```
Comment (Bopzor) [inline apps/server/src/events/event.e2e.spec.ts:66]: **question**

any reason why not deleting this commenting code?

File: apps/server/src/notifications/commands/mark-notification-as-read.command.ts [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
 
-  constructor(private readonly notificationRepository: NotificationRepository) {}
+  constructor(
+    private readonly dateAdapter: DatePort,
+    private readonly database: Database,
+  ) {}
 
   async handle({ memberId, notificationId }: MarkNotificationAsReadCommand): Promise<void> {
-    const notification = await this.notificationRepository.getNotification(notificationId);
+    const notification = await this.database.db.query.notifications2.findFirst({
```
Comment (Bopzor) [inline apps/server/src/notifications/commands/mark-notification-as-read.command.ts:25]: **question**

are we keeping `notification2`?

**Comment by nilscox** [comment]
Thanks for reviewing this draft PR :smile:

File: apps/server/src/notifications/commands/mark-notification-as-read.command.ts (hunk 3 above)
Comment (nilscox) [reply apps/server/src/notifications/commands/mark-notification-as-read.command.ts:25]: Nop, I need to merge all migrations into a single one.

File: apps/server/src/events/event.e2e.spec.ts (hunk 2 above)
Comment (nilscox) [reply apps/server/src/events/event.e2e.spec.ts:66]: Yes, we should uncomment it. I commented it when the query was using the old table, and probably forgot to uncomment it.

File: apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts (hunk 1 above)
Comment (nilscox) [reply apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts:33]: s/suggestion/isue/

File: apps/server/src/events/event.e2e.spec.ts (hunk 2 above)
Comment (nilscox) [reply apps/server/src/events/event.e2e.spec.ts]: fc82172

File: apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts (hunk 1 above)
Comment (nilscox) [reply apps/server/src/events/event-handlers/notify-event-comment-created.handler.ts]: fc82172

File: apps/server/src/notifications/commands/mark-notification-as-read.command.ts (hunk 3 above)
Comment (nilscox) [reply apps/server/src/notifications/commands/mark-notification-as-read.command.ts:25]: a70000e

**Review by Bopzor — APPROVED** [review]

**Pull request merged**